src/commands/register.ts

```diff
@@ -420,4 +424,40 @@ export default class ComponentRegister extends BaseCommand {
   public static getTagFromFlags(flags: any): string {
     return flags.environment ? `${ENV_TAG_PREFIX}${flags.environment}` : flags.tag;
   }
+
+  private async generateDependenciesWarnings(component_dependencies: Dictionary<string>, account_name: string) {
+    const dependency_arr: string[] = [];
+    for (const [component_name, tag] of Object.entries(component_dependencies)) {
+      dependency_arr.push(`${component_name}:${tag}`);
+    }
+    const dependencies: Dictionary<{ component: boolean, component_and_version: boolean }> = (await this.app.api.get(`accounts/${account_name}/components-tags`, { params: { components: dependency_arr } })).data;
+
+    const component_warnings: string[] = [];
+    const tag_warnings: string[] = [];
+    for (const [component_name, valid] of Object.entries(dependencies)) {
+      if (valid.component && valid.component_and_version) {
+        continue;
```

Comment:
Why does the `continue` need to exist? Can't the if statements just be

```ts
if (!valid.component && !valid.component_and_version) {
  component_warnings.push(component_name);
} else if (valid.component && !valid.component_and_version) {
  tag_warnings.push(component_name);
}
```

Reply:
Yes, it can. I was overthinking it.